Makefile

```diff
@@ -112,7 +113,6 @@ ifneq ($(filter release,$(TAGS)),)
 CFLAGS += -O3
 else ifneq ($(filter debug,$(TAGS)),)
 CFLAGS += -O0
-WREN_PARAMS += -DDEBUG=1
```

Comment:
Oh, I didn't realise this line was here. In which case we may be alright, regardless of this change. I should have double-checked.

Reply:
Because It wasn't there the last time you looked :laughing:  I just added it in 7d03937 but changed it in 179456c. I think it makes sense where it is now.